<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registros</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(64, 129, 191, 255), stop:0.524752 rgba(64, 184, 191, 255));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="header">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:none;</string>
     </property>
     <property name="text">
      <string>ACUPUNATOR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="card">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>761</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color:rgb(255, 255, 255);
border-radius:10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Secci√≥n de registros</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Alumnos</string>
         </attribute>
         <widget class="QLineEdit" name="boleta_input">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>139</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>46</y>
            <width>110</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Nombre alumno</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>170</y>
            <width>107</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Asignar a grupo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QComboBox" name="grupos_box">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>170</y>
            <width>141</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="maxVisibleItems">
           <number>100</number>
          </property>
          <item>
           <property name="text">
            <string>das</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nuevo elemento</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>139</y>
            <width>124</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>N√∫mero de boleta</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>108</y>
            <width>119</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Apellido materno</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="nombre_input">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>46</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>77</y>
            <width>115</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Apellido paterno</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="appa_input">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>77</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="apma_input">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>108</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>320</y>
            <width>184</width>
            <height>43</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="regresar_alu_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(164, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Regresar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="guardar_alu_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="alert_alumno">
          <property name="geometry">
           <rect>
            <x>205</x>
            <y>260</y>
            <width>321</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Grupos</string>
         </attribute>
         <widget class="QLabel" name="alert_grupo">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>220</y>
            <width>291</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="grupo_input">
          <property name="geometry">
           <rect>
            <x>221</x>
            <y>160</y>
            <width>291</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>100</y>
            <width>194</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Escriba el nombre del grupo.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>290</y>
            <width>184</width>
            <height>43</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="regresar_gru_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(164, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Regresar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="guardar_gru_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
